### PR TITLE
Release 3.5.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.5.1](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.5.1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.5.1 (2020-02-10)
+
+- Fixed the grid component to properly watch for undefined columns when attempting to get the visual styles for columns. [#96](https://github.com/blackbaud/skyux-grids/pull/96)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the grid component to properly watch for undefined columns when getting a column's visual styles.

That work?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: That's fine :)

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I made the changes.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.5.1](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.5.1)

**Pull request merged**